### PR TITLE
Fix disk configuration values for the different deployment sizes

### PR DESCRIPTION
**Pull request by arbulu89**
The disk configuration is wrong in the `Small` size.
This fixes the error.

**Edit**:
There were more errors rather than only the `Small` size. Commit amended to fix all of them

**FYI**: @petersatsuse

### CODE REVIEW

File: vendor/sources/main.tf [hunk 1]
```diff
@@ -18,11 +18,11 @@ locals {
       vm_size                       = "Standard_E8s_v3"
       enable_accelerated_networking = false
       data_disks_configuration = {
-        disks_type       = "Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS"
-        disks_size       = "128,128,128,128,128,128,128"
-        caching          = "ReadOnly,ReadOnly,None,None,ReadOnly,ReadOnly,ReadOnly"
-        writeaccelerator = "false,false,false,false,false,false,false"
-        luns             = "0,1#2,3#4#5#6#7"
+        disks_type       = "Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS"
```
Comment (smpsgill) [inline vendor/sources/main.tf:21]: Whats the last disk added for?

File: vendor/sources/main.tf [hunk 2]
```diff
@@ -51,9 +51,9 @@ locals {
         caching          = "ReadOnly,ReadOnly,ReadOnly,ReadOnly,ReadOnly,None"
         writeaccelerator = "false,false,false,false,false,false"
         luns             = "0,1,2#3#4#5"
-        names            = "data#log#shared#usrsap#backup"
+        names            = "datalog#shared#usrsap#backup"
```
Comment (smpsgill) [inline vendor/sources/main.tf:54]: Should the data and logs names be separate here? as they have different paths: `/hana/data` and `/hana/log`

**Review by smpsgill — COMMENTED** [review]
@arbulu89 please see comments below

File: vendor/sources/main.tf (hunk 1 above)
Comment (arbulu89) [reply vendor/sources/main.tf:21]: @petersatsuse confirmed me that the luns configuration is `"0,1#2,3#4#5#6#7"`, so this asks for 8 disks

File: vendor/sources/main.tf (hunk 2 above)
Comment (arbulu89) [reply vendor/sources/main.tf:54]: No, this is on purpose. They create a unique `volume group` that later is splited in `70/30` space to provide for `/hana/data` and `/hana/log` respectively.

**Review by smpsgill — APPROVED** [review]

**Pull request merged**